workflows/edit_eml/edit_spatialdata.Rmd

```diff
@@ -63,3 +72,16 @@ Here is an example of what `spatialVector` metadata should look like, including
   <geometry>Polygon</geometry>
 </spatialVector>
 ```
+
+### spatialRasters
+
+Most often these come in GeoTiff or Tiff files. The data is presented as a grid of "pixels". For more information ESRI has a indepth article [here](https://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/what-is-raster-data.htm).
+
+To use the helper function get:
+
+1. the path of your raster file
+2. an attribute table
+
```

Comment:
```suggestion
3. a coordinate system name

To get a coordinate system name, you can use the output of the function on your first try (which will print the coordinate reference system, if it is defined). You can use the return value of `get_coord_list()` (a large `data.frame`) to find the correct coordinate system name.
```

We should explain more about coordinate system names in the spatial data section overall though, since it is required for both spatial vectors and rasters